Tests/IntegrationTests/PropagationTests.hpp

```diff
@@ -117,9 +117,10 @@ inline Acts::NeutralCurvilinearTrackParameters makeParametersCurvilinearNeutral(
 /// \warning Does not check that they are defined on the same surface.
 template <typename charge_t>
 inline void checkParametersConsistency(
-    const Acts::SingleTrackParameters<charge_t>& cmp,
-    const Acts::SingleTrackParameters<charge_t>& ref, double epsPos,
-    double epsDir, double epsMom) {
+    const Acts::SingleBoundTrackParameters<charge_t>& cmp,
+    const Acts::SingleBoundTrackParameters<charge_t>& ref,
+    const Acts::GeometryContext& geoCtx, double epsPos, double epsDir,
```

Comment:
Usually we have GeometryContext as first argument, but this is a test file, so no problem.

Reply:
Unrelated to this MR.